src/test/java/org/jboss/xavier/analytics/test/Utils.java

```diff
@@ -91,8 +92,25 @@ public static void verifyRulesFiredNames(AgendaEventListener agendaEventListener
 
     public static void checkLoadedRulesNumber(StatelessKieSession kieSession, String kiePackageName, int expectedLoadedRules)
     {
-        KiePackage kiePackage = kieSession.getKieBase().getKiePackage(kiePackageName);
-        Assert.assertNotNull("No rules have been loaded from '" + kiePackageName + "' package", kiePackage);
-        assertEquals("Wrong number of rules loaded", expectedLoadedRules, kiePackage.getRules().size());
+        int actualLoadedRules;
+        if (kiePackageName.endsWith(".*"))
+        {
+            final String baseKiePackageName = kiePackageName.substring(0, kiePackageName.length() - 2);
+            Collection<KiePackage> kiePackages = kieSession.getKieBase().getKiePackages();
+            Assert.assertNotNull("No rules have been loaded from '" + kiePackageName + "' package", kiePackages);
```

Comment:
@mrizzi just a suggestion, I think we can use different assert messages for line 100 and 101
```suggestion
            Assert.assertNotNull("No packages have been found in kieSession", kiePackages);
```

Reply:
Thanks @carlosthe19916, good idea :+1: